.github/workflows/pr.yaml

```
@@ -39,6 +40,15 @@ jobs:
     uses: rapidsai/shared-workflows/.github/workflows/conda-cpp-tests.yaml@branch-24.12
     with:
       build_type: pull-request
+  conda-java-tests:
+    secrets: inherit
```

This job should run _after_ the libkvikio conda packages are built, [since it uses those](https://github.com/rapidsai/kvikio/pull/396/files#diff-76d7916ab418397eef76faf7d4eb5f000cff2b417def935c97f0d2c727afb6afR26-R28).
```suggestion
    needs: conda-cpp-build
    secrets: inherit
```